packages/combo-box/test/keyboard.test.js

```diff
@@ -184,220 +184,265 @@ describe('keyboard', () => {
   });
 
   describe('selecting items', () => {
-    beforeEach(async () => {
-      comboBox.value = 'bar';
+    describe('auto-open', () => {
+      beforeEach(async () => {
+        comboBox.value = 'bar';
 
-      comboBox.focus();
-      comboBox.open();
-      await aTimeout(1);
-    });
+        comboBox.focus();
+        comboBox.open();
+        await aTimeout(1);
+      });
 
-    it('should select focused item with enter', async () => {
-      arrowDownKeyDown(input);
-      await aTimeout(1);
-      enterKeyDown(input);
-      await aTimeout(1);
-      expect(comboBox.value).to.equal('baz');
-    });
+      it('should select focused item with enter', async () => {
+        arrowDownKeyDown(input);
+        await aTimeout(1);
+        enterKeyDown(input);
+        await aTimeout(1);
+        expect(comboBox.value).to.equal('baz');
+      });
 
-    it('should clear the selection with enter when input is cleared', () => {
-      filter('');
-      enterKeyDown(input);
+      it('should clear the selection with enter when input is cleared', () => {
+        filter('');
+        enterKeyDown(input);
 
-      expect(comboBox.value).to.eql('');
-    });
+        expect(comboBox.value).to.eql('');
+      });
 
-    it('should close the overlay with enter when custom values are allowed', () => {
-      comboBox.allowCustomValue = true;
-      filter('foobar');
+      it('should close the overlay with enter when custom values are allowed', () => {
+        comboBox.allowCustomValue = true;
+        filter('foobar');
 
-      enterKeyDown(input);
+        enterKeyDown(input);
 
-      expect(comboBox.value).to.equal('foobar');
-      expect(comboBox.opened).to.equal(false);
-    });
+        expect(comboBox.value).to.equal('foobar');
+        expect(comboBox.opened).to.equal(false);
+      });
 
-    it('should stop propagation of the keyboard enter event', () => {
-      const keydownSpy = sinon.spy();
-      document.addEventListener('keydown', keydownSpy);
-      enterKeyDown(input);
-      expect(keydownSpy.called).to.be.false;
-    });
+      it('should stop propagation of the keyboard enter event when dropdown is opened', () => {
+        const keydownSpy = sinon.spy();
+        document.addEventListener('keydown', keydownSpy);
+        enterKeyDown(input);
+        expect(keydownSpy.called).to.be.false;
+      });
 
-    it('should not close the overlay with enter when custom values are not allowed', () => {
-      filter('foobar');
+      it('should not stop propagation of the keyboard enter event when input value is invalid', () => {
```

Comment:
This is the single new test in the new describe block for when auto open is enabled.